implementation/choreography-to-orchestration/README.md

```diff
@@ -0,0 +1,131 @@
+# Convert Choreography to Orchestration
+This project is the CDK implementation of ['Convert Orchestration to Choreography'](https://serverlessland.com/refactoring-serverless/choreography-to-orchestration) pattern. It shows how you can convert a distributed message flow Choreography architecture to a central workflow Orchestration architecture. 
+
+## How it works: 
+The use case is for an application that place an order for a product (Order Placement Workflow). Customer sends a request to place an order for a product. The application completes a series of steps reuired to complete an order and ship the product to customer. 
```

Comment:
spell check - required instead of "reuired".

Reply:
Fixed both issues